modules/op-release-notes-1-11.adoc

```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
 
 * Before this update, a long pipeline run would be stuck in the running state on the cluster, even after the timeout. This update fixes the issue.
 
-* This update fixes the `tkn pr delete` command for correctly using the `keep` flag. Now, if the value of the `keep` flag equals the number of associated task runs or pipeline runs, the `tkn pr delete` command returns exit code `0` along with a message.
\ No newline at end of file
+* This update fixes the `tkn pr delete` command for correctly using the `keep` flag. Now, if the value of the `keep` flag equals the number of associated task runs or pipeline runs, the `tkn pr delete` command returns exit code `0` along with a message.
+
+[id="release-notes-1-11-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.11.1
+
+With this update, {pipelines-title} General Availability (GA) 1.11.1 is available on {product-title} 4.12 and later versions.
+
+[id="fixed-issues-1-11-1_{context}"]
+=== Fixed issues
+
+* Before this update, a task run could fail with a mount path error message, when a running or pending pod was preempted. With this update, a task run does not fail when the cluster causes a pod to be deleted and re-created.
+
+* Before this update, a shell script in a task had to be run as root. With this update, the shell script image has the non-root user ID set so that you can run a task that includes a shell script, such as the `git-clone` task, as a non-root user within the pod.
+
+* Before this update, in {pipelines-title} 1.11.0, when a pipeline run is defined using {pac}, the definition in the Git repository references the `tekton.dev/v1beta1` API version and includes a `spec.pipelineRef.bundle` entry, the `kind` value was sometimes wrongly set to `Task`. The issue did not exist in previous versions of {pipelines-title}. With this update, the `kind` value is set correctly.
```

Comment:
the `kind` value is getting set everytime, and during creating resources on cluster after conversion to v1